app/views/concerns/facet_entry_point_displaying_view.rb

```diff
@@ -49,4 +49,11 @@ def facet_entry_field_title(facet_link_group)
     presenter = FacetPresenter.build(ff, controller)
     presenter.facet_title || facet_link_group.facet_field
   end
+
+  def facet_entry_label(entry)
```

Comment:
@lbiedinger it seems quite wasteful to initialize `Europeana::Blacklight::Response::Facets::FacetField` and `FacetPresenter` for each entry when they are already grouped in this concern by facet field.

Do you think we could/should create one instance of each of those classes for each group of facet entries and extract each entry label from the one call to `FacetPresenter#display`?

Or even create a new method on `FacetPresenter` sepcifically for the content required here by facet entries?